src/pyodide/internal/workers.py

```diff
@@ -25,6 +27,8 @@ class FetchKwargs(TypedDict, total=False):
     method: HTTPMethod = HTTPMethod.GET
 
 
+# TODO: Pyodide's FetchResponse.headers returns a dict[str, str] which means
+#       duplicates are lost, we should fix that so it returns a http.client.HTTPMessage
```

Comment:
Do you still think we should do this? I thought the duplicate values are combined with a comma. How does `http.client.HTTPMessage` handle values with a comma in them?

Reply:
It's not super important, but it would be nice. Splitting values by comma is pretty bleh as far as API design.

But I guess on the other hand that is what JS' Headers expects so maybe it's fine.